unit/Makefile

```diff
@@ -28,6 +28,7 @@ SRC += unit_tests.cpp \
        java_bytecode/inherited_static_fields/inherited_static_fields.cpp \
        pointer-analysis/custom_value_set_analysis.cpp \
        sharing_node.cpp \
+       sharing_map.cpp \
```

Comment:
As @hannes-steffenhagen-diffblue said: these should likely go in the `util` folder.